tests/src/test/scala/common/WskTestHelpers.scala

```diff
@@ -31,6 +31,55 @@ import spray.json._
 import java.time.Instant
 
 /**
+ * An arbitrary response of a whisk action. Includes the result as a JsObject as the
+ * structure of "result" is not defined.
+ */
+case class ActivationResponse(result: Option[JsObject], status: String, success: Boolean)
```

Comment:
remind me why result is an option.
can you add java/scala doc for the parameters.